test/functional/test_framework/test_framework.py

```diff
@@ -740,6 +740,30 @@ def check_sporks_same():
             return all(node.spork('show') == sporks for node in self.nodes[1:])
         wait_until(check_sporks_same, timeout=timeout, sleep=0.5)
 
+    def wait_for_quorum_connections(self, expected_connections=2, timeout = 30):
+        def check_quorum_connections():
+            all_ok = True
+            for node in self.nodes:
+                s = node.quorum("dkgstatus")
+                if s["session"] == {}:
+                    continue
+                if "quorumConnections" not in s:
+                    all_ok = False
+                    break
+                s = s["quorumConnections"]
+                if "llmq_test" not in s:
```

Comment:
I guess this needs #3269 to be merged first

Reply:
oh..yepp..this PR came out of one big branch and I didn't notice this depenency. I'll rebase this later